Formula/libzt.rb

```diff
@@ -0,0 +1,32 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :tag   => "1.3.1"
```

Comment:
Still need a `:revision` argument here to ensure the tag doesn't change.

Reply:
Added